docs/examples/simple_user_model.rst

```diff
@@ -99,8 +99,8 @@ data points)::
 
    >>> print(d)
    name      = NaNO_3
-   x         = [0, 2, 4, 6, 8, 10, 12, 14, 16, 18, 20]
-   y         = [1, 4.125, 7.5, 10.0417, 12.25, 13.75, 14.0833, 14.7917, 15.5, 14.6667, 14.875]
+   x         = Int64[11]
+   y         = Float64[11]
```

Comment:
Does this no cause inconsistency with the line below if you only set a single value?

Reply:
I don't understand the question. This is just a copy of the screen output of the current behavior. Why the staterror isn't converted to a NumPy array but x and y are is a question for a different time/issue.

Reply:
Same as above.